apps/mull-ui/src/utilities.ts

```diff
@@ -83,6 +84,16 @@ export const hasEmoji = (text: string) => {
   return !!emojiRegexRGI().exec(text);
 };
 
+const textLength = string.length;
+export const exceedsNbOfLines = (text: string) => {
+  let countLines = 1;
+  for (let i = 0; i < textLength; i++) if (text.charAt(i) === '\n') countLines++;
+  if (countLines > 6) {
```

Comment:
Why is this set to 6? and what exactly do lines mean as i put in 6 rows of @ and it did not reject me as it wasnt at the char limit 

![image](https://user-images.githubusercontent.com/35787279/113793526-7dc3f100-9716-11eb-86e9-382d0832411a.png)

Reply:
6 is the default number we came up with that seemed reasonable both on mobile and desktop. 
And 6 lines means 6 keyboard `enter`, not rows.